v4l2.c

```diff
@@ -454,12 +454,16 @@ static int set_control(int video_fd, int request_fd, unsigned int id,
 	}
 
 	if (!quiet)
-		fprintf(stderr, "Initialized structure 'controls': (control.id: %d, size: %d)\n",
+	  	/* Ref: https://www.kernel.org/doc/html/v4.10/media/uapi/v4l/vidioc-g-ext-ctrls.html */
+	  	fprintf(stderr, "Initialized 'controls' structure (ctrl_class: %d, which: %d)\n",
+			controls.ctrl_class, controls.which);
+		fprintf(stderr, " 'control' value (id: %d, size: %d)\n",
```

Comment:
That feels overkill, I don't think I'll take it in, sorry.

Reply:
keep it in my branch. For testing